ansible/playbooks/roles/common/tasks/main.yml

```diff
@@ -70,22 +70,23 @@
 
 - name: Disable SELinux at next reboot
   selinux:
-    state: disabled
+    state: disabled # noqa ignore-errors
```

Comment:
Why like that? Are there other options?

Reply:
because we have here ignore_errors: true (line 74). I don't want to change the logic in this PR